demo/src/screens/incubatorScreens/IncubatorTextFieldScreen.tsx

```diff
@@ -191,7 +191,7 @@ export default class TextFieldScreen extends Component {
 
           <TextField
             label="Email"
-            labelColor={{default: Colors.grey10, focus: Colors.blue20, error: Colors.red30, disabled: Colors.grey40}}
+            labelColor={{default: Colors.$textDefault, focus: Colors.$textGeneral, disabled: Colors.$textNeutralLight}}
```

Comment:
I see you're using `textNeutralLight` for `disabled`, it's not very "readable" IMO

Reply:
changed to `$textDisabled`